templates/books-index.html

```diff
@@ -0,0 +1,94 @@
+<div class="container">
+  <div class="row">
+      <div class="col-md-10 col-md-offset-1">
+
+        <form class="form-horizontal" ng-submit="booksIndexCtrl.createBook()">
+          <fieldset>
+
+            <!-- Form Name -->
+            <legend>Add New Book</legend>
+
+            <!-- Text input-->
+            <div class="form-group">
+              <label class="col-md-4 control-label">Book Title</label>
+              <div class="col-md-4">
+              <input id="title" ng-model="booksIndexCtrl.newBook.title" type="text" class="form-control input-md" required="">
+              </div>
+            </div>
+
+            <!-- Text input-->
+            <div class="form-group">
+              <label class="col-md-4 control-label">Author</label>
+              <div class="col-md-4">
+              <input ng-model="booksIndexCtrl.newBook.author" type="text" class="form-control input-md">
+              </div>
+            </div>
+
+            <!-- Text input-->
+            <div class="form-group">
+              <label class="col-md-4 control-label">Release Date</label>
+              <div class="col-md-4">
+              <input ng-model="booksIndexCtrl.newBook.releaseDate" type="text" class="form-control input-md">
+              </div>
+            </div>
+
+            <!-- Text input-->
+            <div class="form-group">
+              <label class="col-md-4 control-label">Book Cover Image</label>
+              <div class="col-md-4">
+              <input ng-model="booksIndexCtrl.newBook.image" type="text" class="form-control input-md">
+              </div>
+            </div>
+
+
+            <!-- Button -->
+            <div class="form-group">
+              <label class="col-md-4 control-label">Save Book</label>
+              <div class="col-md-4">
+                <button class="btn btn-primary">Submit</button>
+              </div>
+            </div>
+          </fieldset>
+        </form>
+      </div>
+  </div>
+
+
+  <input type="text" class="form-control" ng-model="searchText" placeholder="Search Books">
```

Comment:
Using placeholder instead of labels is bad. See https://www.nngroup.com/articles/form-design-placeholders/